scripts/check_configs.ts

```diff
@@ -100,6 +101,14 @@ interface PerpPoolConfig {
   banners: PerpPoolBanner[]
 }
 
+interface MarketBanner {
+  market_id: string;
+  show_from: string;
+  show_until: string;
+  content: string;
+  hideable: boolean;
+}
```

Comment:
According to the struct as defined on L403 of `config.schema.json`, `hideable`, `show_from` and `show_until` are optional fields, which makes this struct inconsistent.

Pls either:
1. Make `hideable`, `show_from` and `show_until` optional in this struct
2. Make `hideable`, `show_from` and `show_until` required on `config.schema.json`